src/start/RealmScreen.js

```diff
@@ -85,10 +101,16 @@ class RealmScreen extends PureComponent<Props, State> {
         {error && <ErrorMsg error={error} />}
         <ZulipButton
           style={styles.smallMarginTop}
-          text="Enter"
+          text="+ Add"
           progress={progress}
           onPress={this.tryRealm}
         />
+        <WebLink
+          style={customStyles.newOrgLink}
+          labelStyle={customStyles.newOrgLinkLabel}
```

Comment:
Creating a `wrapper` for adding marginTop & centre will be better, instead of passing styles object.

Reply:
Not sure if I agree. This looks ok.

Reply:
@jainkuniya @borisyankov OK, for now I'll keep this as-is. Please let me know if there is anything else I should change. Thanks!